src/main/asciidoc/inc/image/_naming.adoc

```diff
@@ -50,6 +50,9 @@ When specifying the container name pattern the following placeholders can be use
 | *%i*
 | An index which is incremented if a container has already been created. With this parameter it is easily possible to have multiple, similar containers. See the example below for more details.
 
+| *%r*
+| A random container name choosen by the docker engine.
```

Comment:
```suggestion
| An empty name that triggers are random container name chosen by the docker engine. If using `%r` this must be the only content of this pattern. Otherwise, an error is thrown.
```